scripts/bump.js

```diff
@@ -1,5 +1,5 @@
 #!/usr/bin/env node
-
+/* eslint-disable array-callback-return */
```

Comment:
Calling out this specific disable I had to add to avoid changing some `.map()` calls below